main.py

```diff
@@ -2,6 +2,68 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    def create(
```

Comment:
I didn't expect you to define these functions inside main(), it is possible, but it would make more sense to have them defined above main